utils/frame/benchmarking-cli/src/storage/write.rs

```diff
@@ -62,49 +68,102 @@ impl StorageCmd {
 		let (mut rng, _) = new_rng(None);
 		kvs.shuffle(&mut rng);
 
+		let mut child_tries = Vec::new();
```

Comment:
Applied requested changes.
I didn't want to increase memory usage at the time, but I'm going to open another PR later to add randomized sampling to avoid loading all data in memory.